src/main/io/gps.c

```diff
@@ -1561,6 +2099,11 @@ static union {
     ubxNavSat_t sat;
     ubxCfgGnss_t gnss;
     ubxAck_t ack;
+    struct {
+        char swVersion[30];
+        char hwVersion[10];
+        char extension[10][30];
+    } ver;
```

Comment:
This should be separate struct, like other members

Reply:
we aren't using `extension` for any purpose, so at this point it doesn't need to be a separate struct at this point.

Reply:
In a recent commit, I did fix up the `ubxMonVer_t` typedef to be accurate, and replaced the struct definition in the `_buffer` union to use that be consistent with the other payloads contained in it.

So this is resolved as suggested.